rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2842,6 +2880,34 @@ def _check_tokens_existence(user_db_cursor: 'DBCursor', expect_removed: bool = F
 
         _check_tokens_existence(user_db_cursor=cursor, expect_removed=True)
 
+        assert cursor.execute(  # events with the correct prefix for coinbase removed
+            'SELECT COUNT(*) FROM history_events WHERE location=? AND event_identifier LIKE ?',
+            (coinbase_location, f'{CB_EVENTS_PREFIX}%'),
+        ).fetchone()[0] == 0
+
+        assert cursor.execute(  # events imported or different locations preserved
+            'SELECT COUNT(*) FROM history_events WHERE event_identifier IN (?, ?)',
+            ('kraken-id', f'{ROTKI_EVENT_PREFIX}_EVENT'),
+        ).fetchone()[0] == 2
+
+        assert cursor.execute(  # all coinbase caches are deleted
+            'SELECT COUNT(*) FROM key_value_cache WHERE name LIKE ? OR name LIKE ?',
+            (f'{(coinbase_loc := Location.COINBASE.serialize())}_%_last_query_ts', f'{coinbase_loc}_%_last_query_id'),  # noqa: E501
+        ).fetchone()[0] == 0
+        assert cursor.execute(  # check that the kraken cache was not affected
+            'SELECT COUNT(*) FROM key_value_cache WHERE name LIKE ? OR name LIKE ?',
+            (f'{(kraken_loc := Location.KRAKEN.serialize())}_%_last_query_ts', f'{kraken_loc}_%_last_query_id'),  # noqa: E501
+        ).fetchone()[0] == 1
+        assert cursor.execute(  # ensure trades with Coinbase location and a link are deleted
+            'SELECT COUNT(*) FROM trades WHERE location=? AND link != ?',
+            (coinbase_location, ''),
+        ).fetchone()[0] == 0
+        assert cursor.execute('SELECT COUNT(*) FROM trades').fetchone()[0] == 2  # ensure trade with empty link is preserved  # noqa: E501
```

Comment:
shouldn't you have a more exact query here? Not just the count. But exactly which trades survived.